File: app/src/main/java/com/OdiousPanda/rainbow/API/AQICall.java
package com.OdiousPanda.rainbow.API;

import com.OdiousPanda.rainbow.DataModel.AQI.AirQuality;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface AQICall {
    @GET("feed/geo:{lat};{lon}/?token=" + Constant.AQI_API_KEY)
    Call<AirQuality> getAirQuality(@Path("lat") String lat, @Path("lon") String lon);
}

File: app/src/main/java/com/odiousPanda/rainbowKt/apis/AQICall.kt
package com.odiousPanda.rainbowKt.apis

import com.odiousPanda.rainbowKt.model.dataSource.aqi.Aqi
import retrofit2.Response
import retrofit2.http.GET
import retrofit2.http.Path

interface AQICall {
    @GET("feed/geo:{lat};{lon}/?token=" + APIConstants.AQI_API_KEY)
    suspend fun getAirQuality(
        @Path("lat") lat: String,
        @Path("lon") lon: String
    ): Response<Aqi>
}